python/pyspark/sql/functions.py

```diff
@@ -1921,10 +1921,12 @@ def from_json(col, schema, options={}):
 @since(2.1)
 def to_json(col, options={}):
     """
-    Converts a column containing a [[StructType]] or [[ArrayType]] of [[StructType]]s into a
-    JSON string. Throws an exception, in the case of an unsupported type.
+    Converts a column containing a [[StructType]], [[ArrayType]] of [[StructType]]s,
+    a [[MapType]] or [[ArrayType]] of [[MapType]] into a JSON string.
+    Throws an exception, in the case of an unsupported type.
```

Comment:
While we are here, let's fix `[[StructType]]` to `` :class:`StructType` `` (and the same instances too) to make Python API documentation pretty.

Reply:
ok Thanks.